### PR TITLE
create-to-do

### PR DESCRIPTION
**Pull request by biararaujo**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,27 +1,14 @@
...
-  const handleClick = () => {
-    // TODO
-  };
-
   return (
-    <div className="app">
-      <div>Header</div>
-      <div>Content</div>
-      <div>
-        {items.map((item: string) => (
```
Comment (jhrodriguesit) [inline src/App.tsx:22]: Great work Bia.
I think if we delete this iteration we will stop showing the todo list in the screen. We want to keep showing the list.

File: src/index.tsx [hunk 2]
```diff
@@ -3,16 +3,15 @@ import ReactDOM from "react-dom/client";
 import "./index.scss";
 import App from "./App";
 import reportWebVitals from "./reportWebVitals";
+import CreateToDo from "./components/CreateToDo/CreateToDo";
```
Comment (jhrodriguesit) [inline src/index.tsx]:
```suggestion

```
This import should be removed as this component is not being used here.

File: src/index.tsx [hunk 3]
```diff
@@ -3,16 +3,15 @@ import ReactDOM from "react-dom/client";
...
 const root = ReactDOM.createRoot(
   document.getElementById("root") as HTMLElement
 );
 
-root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
-);
+root.render(<React.StrictMode>
```
Comment (jhrodriguesit) [inline src/index.tsx]: Are you using the `Prettier` vscode extension? It looks like your your code is not indented properly.

**Review by jhrodriguesit — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (biararaujo) [reply src/App.tsx:22]: Okay, João! I put it again.

File: src/index.tsx (hunk 2 above)
Comment (biararaujo) [reply src/index.tsx]: Done!

File: src/components/CreateToDo/CreateToDo.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+import "./CreateToDo";
+import "./CreateToDo.scss";
+import { BiPlusCircle } from "react-icons/bi";
+
+const CreateToDo = ({
+  onAdd,
+  stopAdd,
+}: {
+  onAdd: () => void;
+  stopAdd: () => boolean;
```
Comment (jhrodriguesit) [inline src/components/CreateToDo/CreateToDo.tsx]: Good work! But I am not sure if we need to pass this function as a props for this component. Can we try to do it in a simpler way?

File: src/components/CreateToDo/CreateToDo.tsx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    setIsHovered(true);
+  };
+
+  const handleMouseLeave = () => {
+    setIsHovered(false);
+  };
+
+  return (
+    <div>
+      {/* {hasNull ? (
```
Comment (jhrodriguesit) [inline src/components/CreateToDo/CreateToDo.tsx]: Could you please remove this comment section?

**Review by jhrodriguesit — CHANGES_REQUESTED** [review]
Could we please resolve the conflict and ensure that this changes are working fine with the latest main branch?

File: src/components/CreateToDo/CreateToDo.tsx (hunk 4 above)
Comment (lucaspereiraio) [reply src/components/CreateToDo/CreateToDo.tsx]: Already done.

File: src/components/CreateToDo/CreateToDo.tsx (hunk 5 above)
Comment (lucaspereiraio) [reply src/components/CreateToDo/CreateToDo.tsx]: Already done.

**Review by jhrodriguesit — APPROVED** [review]

**Review by Jeffersonlpu — APPROVED** [review]

**Pull request merged**